src/modules/form/components/DynamicField.tsx

```diff
@@ -1,5 +1,5 @@
 import { Stack, Typography } from '@mui/material';
```

Comment:
Once this issue https://github.com/greenriver/hmis-warehouse/pull/2901/files#r1148026223 is resolved, you'll also need to make  a change to line 120 of this file to handle the case when `value` is an array